search/serializers.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import serializers
+from django.db import models
+
+from account.models import (
+    Student,
+    Parent,
+    Instructor
+)
+from course.models import (
+	Course,
```

Comment:
some of these indentation seems off, let's make the indentation for the rest of this file consistent